twisted/names/client.py

```diff
@@ -362,8 +362,8 @@ def filterAnswers(self, message):
         If the message was truncated, re-attempt the query over TCP and return
         a Deferred which will fire with the results of that query.
 
-        If the message's result code is not L{dns.OK}, return a Failure
-        indicating the type of error which occurred.
+        If the message's result code is not C{twisted.names.dns.OK}, return a 
```

Comment:
In future, try to make sure that when you edit lines, you don't add any trailing whitespace (unless it is required in the code....which is very rare).   I can take care of this for now.
Thanks for the submission!